komga-webui/src/components/DropZone.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <label class="drop-zone" v-cloak @drop.prevent="dropHandler" @dragover.prevent>
+    <span class="file-input">Choose an image</span> - drag and drop
+    <input hidden aria-hidden="true" type="file" accept="image/*" multiple @change="dropHandler" >
+  </label>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'DropZone',
+  methods: {
+    dropHandler(event: Event) {
+      if (event instanceof DragEvent && event.dataTransfer) {
+        const droppedFiles = event.dataTransfer.files
+        if (!droppedFiles) return
+
+        this.$emit('on-input-change', Array.from(droppedFiles))
+      }
+      if (event.target instanceof HTMLInputElement && event.target.files) {
+        const selectedFiles = event.target.files
+        if (!selectedFiles) return
+
+        this.$emit('on-input-change', Array.from(selectedFiles))
+      }
+    },
+  },
+})
+</script>
+
+<style scoped>
+.drop-zone {
+  background:repeating-linear-gradient(
+    45deg,
```

Comment:
```suggestion
    135deg,
```

For some reason i prefer the hashing in the other direction